frontend/public/metalkube/components/utils/selectors.tsx

```diff
@@ -0,0 +1,15 @@
+import { defaults } from 'lodash-es';
+
+export const getHostSpec = bmo =>
+  defaults(bmo.spec, {
```

Comment:
i would also prefer `get(bmo, 'spec', {online: false})` as it does not change `bmo` object as @suomiy said. This applies for other selectors too